client/src/ui/molecules/breadcrumbs/index.scss

```
@@ -51,6 +51,10 @@
           outline-offset: 1px;
         }
       }
+      
+      &:hover {
+        background-color: var(--button-secondary-hover);
+      }
```

We discussed this today, but instead of changing the background-color on hover, we would like the breadcrumb to be underlined on hover. Likewise on focus.

Would you mind changing your PR accordingly?

Also, could you move this block after the `:link`/`:visited` block above (~ line 44)?

```suggestion
      &:hover,
      &:focus {
        text-decoration: underline;
      }
```